qlty-check/src/planner/plan.rs

```diff
@@ -27,6 +28,7 @@ pub struct Plan {
     pub transformers: Vec<Box<dyn IssueTransformer>>,
     pub invocations: Vec<InvocationPlan>,
     pub fail_level: Option<Level>,
+    pub config_staging_operations: Vec<ConfigOperation>,
```

Comment:
Would this field be better named `config_operations` or is the "staging" part important?